### PR TITLE
USWDS-Site: Documentation updates for release 3.6.1 

### PR DESCRIPTION
**Pull request by amyleadem**
# Summary

Changelogs for all updates in the USWDS 3.6.1 release.

:warning: We need to update changelog dates and release numbers before merging. 

## Preview link

- [Banner changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.6.1/components/banner/#changelog) 
  - Changelog PR: #2267
  - Feature PR: uswds/uswds#5370
- [Breadcrumb changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.6.1/components/breadcrumb/#changelog) 
  - Changelog PR: #2271
  - Feature PR: uswds/uswds#5497
- [Card changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.6.1/components/card/#changelog) 
  - Changelog PR: #2227
  - Feature PR: uswds/uswds#5423
- [Checkbox changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.6.1/components/checkbox/#changelog) 
  - Changelog PR: #2267
  - Feature PR: uswds/uswds#5494
- [Header changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.6.1/components/header/#changelog) 
  - Changelog PR: #2251
  - Feature PR: uswds/uswds#5443)
- [Modal changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.6.1/components/modal/#changelog) 
  - Changelog PR: #2267
  - Feature PR: uswds/uswds#5493
- [Radio changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.6.1/components/radio-buttons/#changelog) 
  - Changelog PR: #2270
  - Feature PR: uswds/uswds#5494
- [Table changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.6.1/components/table/#changelog) 
  - Changelog PR: #2264
  - Feature PR: uswds/uswds#5477
- [Tooltip changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.6.1/components/tooltip/#changelog) 
  - Changelog PR: #2265
  - Feature PR: uswds/uswds#5475

### CODE REVIEW

File: _data/changelogs/component-checkbox.yml [hunk 1]
```diff
@@ -2,6 +2,14 @@ title: Checkbox
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Updated radio and checkbox tile styling to have lighter borders.
```
Comment (mejiaj) [inline _data/changelogs/component-checkbox.yml:6]: Changing to `styles` could read better.

```suggestion
    summary: Updated radio and checkbox tile styles to have lighter borders.
```

File: _data/changelogs/component-checkbox.yml [hunk 2]
```diff
@@ -2,6 +2,14 @@ title: Checkbox
...
+    summaryAdditional:
```
Comment (mejiaj) [inline _data/changelogs/component-checkbox.yml:7]: Consider note related to what the accessibility issue it affects.

**Review by mejiaj — APPROVED** [review]
Minor suggestion to checkbox & radio; to include note on a11y change. 

Otherwise confirming:

- [x] User facing changes are included.
- [x] PRs numbers are accurate.

File: _data/changelogs/component-checkbox.yml (hunk 2 above)
Comment (mahoneycm) [reply _data/changelogs/component-checkbox.yml:7]: I agree with James. I like when changelogs also address _why_ the change was made.

File: _data/changelogs/component-checkbox.yml (hunk 1 above)
Comment (thisisdano) [reply _data/changelogs/component-checkbox.yml:6]:
```suggestion
    summary: Updated radio and checkbox tiles to have lighter borders, reducing visual noise.
```

File: _data/changelogs/component-checkbox.yml (hunk 2 above)
Comment (thisisdano) [reply _data/changelogs/component-checkbox.yml:7]: Changed to `Updated radio and checkbox tiles to have lighter borders, reducing visual noise.`

**Comment by amyleadem** [comment]
@thisisdano Thanks for making that update for the checkbox/radio changelogs. Everything looks good to me! 

I think the only items left for this are to merge in https://github.com/uswds/uswds-site/pull/2293 and  https://github.com/uswds/uswds-site/pull/2292. Are there any other outstanding items here?

**Comment by thisisdano** [comment]
I believe the only remaining changes are to update to USWDS 3.6.1 when its published and update the release #s and dates in the changelog. I'll update the dates and release numbers now.

**Pull request merged**